tests/vocabulary.test.ts

```diff
@@ -0,0 +1,51 @@
+import VocabularyClient from '../src/VocabularyClient';
+import { mochaAsync } from './mochaAsync';
+import { expect } from 'chai';
+import 'mocha';
+
+describe('Vocabulary tests', function () {
+    let client = new VocabularyClient({ endpoint: global.endpointUrl, key: process.env.NEXOSIS_API_TESTKEY });
+    this.timeout(5000);
+
+    it('can list vocabularies', () => {
+        return client.list().then((data) => {
+            expect(data.items.length).to.be.greaterThan(0);
+        });
+    });
```

Comment:
Minor nitpick on these tests would be to use mochaAsync style tests with async and await.  Makes the tests read a little more clearly since you don't have to chain in promises.

Reply:
I'm game